frontend/src/components/RejectionReason/Reasons/TokenModuleReject.vue

```diff
@@ -207,36 +139,21 @@
 				<span class="px-2">
 					Requested amount :
 					<b>
-						{{
-							// Todo: fix this when we reset the devnet DB
-
-							reason.details[reason.reasonType]?.requested_amount ??
-							reason.details.requested_amount
-						}}
+						{{ reason?.details?.requested_amount }}
```

Comment:
Can these actually be undefined. I want to be sure that we don't display here `undefined` one day. Applies to the other areas as well.